bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TprimeTprime_ttHH_LO/TprimeTprime_ttHH_madspin_card.dat

```diff
@@ -0,0 +1,24 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+
+set ms_dir ./madspingrid
+set max_running_process 1
+
+set Nevents_for_max_weight 25 # number of events for the estimate of the max. weight
```

Comment:
for MG version used in UL, there should be "Nevents_for_max_weigth"

Reply:
Now fixed, thanks!